Group_Constitution

```diff
@@ -0,0 +1,127 @@
+Constitution of “Mozilla Science Study Group- University of Toronto”
+
+1.	Name
+
+The official name of this recognized campus group is “     ”
+
+The acronym or abbreviation of this group is:      
+
+2.	Purpose and Objectives
+
+The purpose, objectives, mission and/or mandate of organization is outlined here:
+     
```

Comment:
Oh sorry, I edited it all in a .doc file so I don't think the changes are
showing! I'll do it in the text file asap.

On Tue, Sep 8, 2015 at 3:08 PM, Luke W Johnston <notifications@github.com>
wrote:

> In Group_Constitution
> <https://github.com/UofTCoders/studyGroup/pull/44#discussion_r38966726>:
>
> > @@ -0,0 +1,127 @@
> > +Constitution of “Mozilla Science Study Group- University of Toronto”
> > +
> > +1.	Name
> > +
> > +The official name of this recognized campus group is “     ”
> > +
> > +The acronym or abbreviation of this group is:
> > +
> > +2.	Purpose and Objectives
> > +
> > +The purpose, objectives, mission and/or mandate of organization is outlined here:
> > +
>
> I made a comment of this in the previous PR for the constitution:
>
> Some suggestions:
>
>
>    - To share and learn about other coding techniques and best practices
>    for computing and analysis
>    - To improve the collective competence and skills of
>    researchers-in-training at the University of Toronto
>    - To build a community of like-minded individuals interested in
>    teaching and learning about coding/programming in the context of scientific
>    research
>    - To increase the awareness and need to learn proper programming and
>    coding principals in order to improve the science that uses these techniques
>
> The last one may just be specific to my field (and in general biomedical
> sciences), but there is a real lack of knowledge about how important
> programming/coding is for research and science... or even a lack of
> understanding that programming exists! (partly in jest)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UofTCoders/studyGroup/pull/44/files#r38966726>.
>



-- 
Lina Tran
MSc. Candidate | Frankland Lab
Department of Physiology
University of Toronto